### ISSUE TITLE
dev/php8 wont connect object_lots and objects through related bundle

### ISSUE THREAD

**Issue by Monica-Wood** [collectiveaccess/providence#1492]
Not sure how long this has been a problem for, but it's happening on all my php8 servers, some are behind by about 14-18 commits, my dev is up to date.

When I go to add an object to the object lot (or vice versa) using the related bundles, you can search for the record, but it wont let you select it.
If you do a quickadd, when you hit save, it forever spins in a 'processing form' 

If adding the object from the object lot 'add new' in the inspector, it is created and relates without any issue.

**Comment by Monica-Wood** [comment]
This is connected with the issue I logged about ca_item_tags.  When linking from object lot side the same jquery error is shown, when linking from object side, the error is slightly different, but same line

```
Uncaught TypeError: Cannot read properties of undefined (reading '64')
    at options.select (ca.relationbundle.js?rev=GIT_2.0:183:45)
```

**Comment by collectiveaccess** [comment]
This is just for tags?

**Comment by Monica-Wood** [comment]
it's for ca_item_tags and object_lot<->object relationships.

**Comment by Monica-Wood** [comment]
Solved with #1493
